### PR TITLE
UPD: Update recipe for thrift 0.11.0 dependency

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

**Comment by cpcloud** [comment]
@xhochy Can you review this when you get a chance?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set version = "1.4.0.pre" %}
-{% set commit = "e2814ccb224a01289ad3a736ce89a956cb7d52fb" %}
-{% set sha256sum = "55a507878f0d53b8b8319f826df2acd9e52655db317ce04329b9ba3c3880d562" %}
+{% set commit = "c064f827e96b37d29ba7a7b87657a18a94896854" %}
+{% set sha256sum = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855d" %}
```
Comment (xhochy) [inline recipe/meta.yaml]: CircleCI fails on the sha256sum

Comment (cpcloud) [reply recipe/meta.yaml]: Yep, fixed it

**Comment by cpcloud** [comment]
Looks like for some reason the autolink disabling patch is causing a failiure

**Comment by cpcloud** [comment]
I removed the patch and updated the commit to the actual parquet-cpp patch that has those changes.

**Review by wesm — COMMENTED** [review]
Can you bump the build number? Otherwise LGTM

**Comment by cpcloud** [comment]
Bumping.

**Comment by cpcloud** [comment]
Merging on green.

**Pull request merged**